### ISSUE TITLE
Spanish Traslations Greenligth 2.7.1 

### ISSUE THREAD

**Issue by UnderFenex** [bigbluebutton/greenlight#2016]
Hi, I am cheking the config/locales/es.yml and es_MX.yml, and they have valid translations for spanish.

However in many places, this traslations are not being shown, and it seems that the english ones are.

**Comment by farhatahmad** [comment]
Can you give me some examples of places its not showing?

**Comment by UnderFenex** [comment]
Yes, I managed to solved it. The issue is that the locale es.yml has missing fields that exist on the en.yml. Adding the fields with the respective translations fixed it.
